controllers/docker/linux/eksa-diagnostic-collector/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+FROM $BASE_IMAGE
+
+RUN yum install tar -y
```

Comment:
lets update this based on https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/aws/eks-anywhere/docker/linux/eksa-diagnostic-collector/Dockerfile#L4.  We also should add `rm -rf /var/cache/yum`